cmd/bridge/main.go

```diff
@@ -241,6 +244,63 @@ func main() {
 		QuickStarts:               *fQuickStarts,
 		AddPage:                   *fAddPage,
 		ProjectAccessClusterRoles: *fProjectAccessClusterRoles,
+		K8sProxyConfigs:           make(map[string]*proxy.Config),
+		K8sClients:                make(map[string]*http.Client),
+	}
+
+	managedClusterConfigs := []*serverconfig.ManagedClusterConfig{}
+	if *fManagedClusterConfigs != "" {
+		unvalidatedManagedClusters := []serverconfig.ManagedClusterConfig{}
+		if err := json.Unmarshal([]byte(*fManagedClusterConfigs), &unvalidatedManagedClusters); err != nil {
+			klog.Errorf("Unable to parse managed cluster JSON: %v", *fManagedClusterConfigs)
```

Comment:
We should use `klog.Fatalf` and fail if the managed cluster configuration is invalid.

Reply:
My thought was that an invalid managed cluster config shouldn't crash the whole console. I guess if you pass an incorrect arg it should fail though. I'll update here and in flag parsing logic.

Reply:
Yeah, I think it's better simply to fail. We need to indicate there's a problem (with more than just a log message). We do a rolling deployment, so console will still be available with the previous config. This way the console operator will reflect a degraded status and trigger an alert.